### PR TITLE
Add function deleteDatabase

### PR DESCRIPTION
**Pull request by kcampion**

### CODE REVIEW

**Comment by cosmith** [comment]
Looks good to me but I don't really have time to test it now, since I don't use Cordova anymore. If you feel that it's good to merge I'll merge it.

**Comment by kcampion** [comment]
I'll finish some tests to use it on my production. After that I'll notify you when it will be ready.

File: src/loki-cordova-fs-adapter.es6 [hunk 1]
```diff
@@ -64,6 +64,24 @@ class LokiCordovaFSAdapter {
             }
         );
     }
+    
+    deleteDatabase(dbname, callback) {
+        window.resolveLocalFileSystemURL(cordova.file.dataDirectory,
+            (dir) => {
+                let fileName = this.options.prefix + "__" + name;
```
Comment (guewen) [inline src/loki-cordova-fs-adapter.es6]: shouldn't `name` be `dbname`?

File: src/loki-cordova-fs-adapter.es6 [hunk 2]
```diff
@@ -64,6 +64,24 @@ class LokiCordovaFSAdapter {
...
+                let file = dir.getFile(fileName, {create: true}, handleSuccess, handleError);
```
Comment (guewen) [inline src/loki-cordova-fs-adapter.es6]: aren't `handleSuccess` and `handleError` undefined?

**Comment by kcampion** [comment]
I've finished the .js file @cosmith. However the .es6 file hasn't be tested cause my compiler create a strange file, even with the source.

**Comment by guewen** [comment]
Tested the `.js` version, I confirm it works.

There is still a typo in the `.es6` version here: `let fileName = this.options.prefix + "__" + name;`, should be `dbname`.

**Comment by kcampion** [comment]
Indeed @guewen, it's fixed.

**Comment by guewen** [comment]
thanks for this proposal @kcampion, it literally saved my day :)

**Comment by cosmith** [comment]
Thank you @kcampion and @guewen ! Merging!

**Comment by guewen** [comment]
FYI I proposed the equivalent in lokijs https://github.com/techfort/LokiJS/pull/329 so we can directly call `db.deleteDatabase()` instead of `db.adapter.deleteDatabase()`. (See also https://github.com/techfort/LokiJS/pull/327#issuecomment-175728141)

**Pull request merged**